tests/functional/wdio.conf.js

```diff
@@ -17,6 +17,12 @@ exports.config = {
   // directory is where your package.json resides, so `wdio` will be called from there.
   //
   specs: ["./tests/functional/spec/**/*.js"],
+  suites: {
+    timeout_modal: ["./tests/functional/spec/timeout/*.spec.js"],
+    components: ["./tests/functional/spec/components/**/*.js"],
+    features: ["./tests/functional/spec/features/**/*.js"],
+    general: ["./tests/functional/spec/general/*.js"],
```

Comment:
I'm not against this, but the only issue is if someone put a new spec into `/tests/functional/spec/new_test.spec.js`, it wouldn't get run, and we wouldn't realise?. Maybe general could just be `/tests/functional/spec/*.spec.js`. Same issue if someone adds a new folder, but that is very unlikely as the folder we have now covered pretty much everything.